middleware/markdown/summary/summary.go

```diff
@@ -0,0 +1,16 @@
+package summary
+
+import (
+	"bytes"
+
+	"github.com/russross/blackfriday"
+)
+
+func Markdown(input []byte, wordcount int) []byte {
+	words := bytes.Fields(blackfriday.Markdown(input, Renderer{}, 0))
+	if wordcount > len(words) {
+		wordcount = len(words)
+	}
+
+	return bytes.Join(words[0:wordcount], []byte{' '})
```

Comment:
I'd prefer to use the provided methods here.  At the very least, using `unicode` to interpret things as runes, not just bytes.